### PR TITLE
Selector dropdowns

### PR DESCRIPTION
**Pull request by hone1er**
### What changes were made?

Created an `<Expertise />` component that handles the functional and industry expertise selection/edit features

### Is there any background info about the change?

- Allows user to select up to 3 functional and/or industry expertise instead of just 1

### Are there any state changes? Mention key ones (if any)

Added a `count` state to track how many selection fields have been created. Max 3

- Default: The number of previously saved "expertise" or 1

Also added the following to the profileData for tracking the expertise selections:
- functionalExpertise2
- functionalExpertise3
- industryExpertise2
- industryExpertise3

### Screenshots or Gifs✨
Screen Shot 2022-03-09 at 11 43 10 AM

Screen Shot 2022-03-09 at 11 43 45 AM

Screen Shot 2022-03-09 at 11 44 15 AM
### Any new dependencies? Why were they added?

none

### Shortcut Link

https://app.shortcut.com/twali/story/267/create-selector-dropdowns-with-tags-for-industry-functional-expertise

### CODE REVIEW

File: components/HeaderNav/HeaderNav.tsx [hunk 1]
```diff
@@ -50,8 +50,12 @@ export interface Identity {
   businessLocation: string;
   currTitle: string;
   currLocation?: string;
-  funcExpertise: string;
+  functionalExpertise: string;
+  functionalExpertise2: string;
+  functionalExpertise3: string;
```
Comment (nagmak) [inline components/HeaderNav/HeaderNav.tsx:55]: Just wondering, if it's possible to use an array here instead... that way we won't have to track 6 different functional and industry expertise values separately!

File: components/SignUpSteps/SignUpSteps.tsx [hunk 2]
```diff
@@ -366,116 +371,108 @@ const professionalProfileStep = ({ handleChange, values, errors }) => {
...
+                "Finance",
+                "Financial Exchanges",
+                "Financial Services",
+                "FinTech",
+                "Fraud Detection",
+                "Funding Platform",
+                "Gift Card",
+                "Health Insurance",
+                "Hedge Funds",
+                "Impact Investing",
```
Comment (nagmak) [inline components/SignUpSteps/SignUpSteps.tsx:454]: also would be better to have these options in a separate constants file like `https://github.com/twali-xyz/twali-xyz/blob/main/utils/countriesConstants.ts` so we can reuse them, and update the values if needed

File: components/HeaderNav/HeaderNav.tsx (hunk 1 above)
Comment (hone1er) [reply components/HeaderNav/HeaderNav.tsx:55]: I tried to figure out how to get them all in an array but due to how our handleChange function works I couldn't really figure it out. Will think about how to make that work

File: components/SignUpSteps/SignUpSteps.tsx (hunk 2 above)
Comment (hone1er) [reply components/SignUpSteps/SignUpSteps.tsx:454]: Will do

**Comment by hone1er** [comment]
@nagmak i got the changes implemented. took me forever to figure out how to get everything stored nicely in an array lol but i think it's working now

i also moved my `setExpertise` function to a new `helpers` folder and created `components` folder in the `Profile` folder. For now the components folder just holds my new `<Expertise />` component but i place other components in the folder as i refactor the Profile page

let me know what you think

**Comment by hone1er** [comment]
Ok got some serious problems that need to be worked out the logic is getting crazy now

**Review by nagmak — APPROVED** [review]
WOOO. Looks good to me! 🔥

**Pull request merged**